CuratedContent/KanbanAndScrumMakingTheBestOfBoth.md

```diff
@@ -0,0 +1,61 @@
+Kanban is likely the least prescriptive framework for lean/agile project/development management that has even been proposed that is based up with solid theory and evidence.
+By incrementally adding some tools from Scrum, Kanban can be extended and modified to address almost any project and create a continuum of lean/agile project management processes from basic Kanban to full-blow Scrum.
+
+Part I of this mini-book is about 50 pages and provides an excellent overview and discussion of Scrum and Kanban and breaks them down into their pieces to come up with a powerful toolbox of building-blocks.
+This material is a fast read and provides the basic terminology for discussion of Kanban, Scrum and agile methods in general.
+Part II of this mini-book is a 50-page case study in the application of these building blocks and experience from a real project.
+Unlike extended examples in some books, the case study presented in Part II really enhances the material covered in Part I.
+
+This mini-book is packed with helpful diagrams and cartoons that convey the concepts being discussed in a very power way.
+Just the diagrams and cartoons alone make this mini-book worth the time to browse and is a great way to refresh these concepts in one's mind.
+
+Considering there are much longer books on lean and agile methods, this mini-book, in my view, provides the best usage of one's time in to get a grip on these concepts with the least amount of investment.
+I can't recommend this mini-book more strongly if one is interested in improving one's software project management processes using lead/agile methods.
+
+The ideas and insights of this mini-book were used as the foundation for the software development management processes in the CSE Consortium For Advanced Simulation Of Light Water Reactors (CASL) project and especially in the Nuclear Reactor Physics Integration Focus area  starting in 2010.
+By using the iterative retrospective process and with experiments, a very successful iteration-based Kanban process was constructed and variations of that process endure in some of the groups that participated in the CASL project after the main 10-year project ended.
+
+At the time of this writing, this mini-book is available for free download from [infoq.com](https://www.infoq.com/mini-books/kanban-scrum-mini-book/) but you must register with infoq.com to get it.
+A printed version of the mini-book that can be mailed to you is also available for a very small fee.
+
+<img src='https://github.com/betterscientificsoftware/images/raw/master/KanbanAndScrumMakingTheBestOfBoth.png' class='logo' />
+
+
+<!---
+Publish: preview
+RSS update: ???
+Categories: Planning, Development, Collaboration
+Topics: Software process improvement, Software engineering, Requirements, Issue tracking, Strategies for more effective teams
+Tags: book
+Level: 2
+Prerequisites: defaults
+Aggregate: none
+--->
+
+
+<!--
```

Comment:
What is this "local words" meta tag? I am not familiar with this

Reply:
> What is this "local words" meta tag? I am not familiar with this

That is how emacs spell-check (aspell) manages local file ignores.  I put this in an HTML comment so it would be hidden.